### DISCUSSION TITLE
[Help Wanted]Export MD file and i dont want it carries some meta data

### DISCUSSION THREAD

**Discussion by LOYINuts** [windingwind/zotero-better-notes#1187] [Q&A] [answered]
When i use this plugin to export my note to md file and I open it using obsidian,it shows like this:
![QQ20241109-173634](https://github.com/user-attachments/assets/760c7ac2-42dc-4bfd-8495-ed186e84e7c5)

and i dont want it carries these info, i didnt find the solution in issues.

**Comment by windingwind** [comment, marked as answer]
<img width="350" alt="image" src="https://github.com/user-attachments/assets/6c73ac46-8b07-4249-9811-0096e9ba8a78">

**Reply by LOYINuts** [reply]
The problem still exist :(

**Reply by LOYINuts** [reply]
oh i find that i should close the set auto-sync function. then it wont show these infos... but i want to use the auto-sync function. is there a solution?

**Reply by windingwind** [reply]
No. This YAML header is required for syncing notes to markdown.

**Reply by LOYINuts** [reply]
OK. really appreciate your help :)
